libs/fixture-generators/src/generate-cvrs/generate_cvrs.ts

```diff
@@ -283,16 +294,14 @@ export function* generateCvrs({
                 CreatingDeviceId: scannerId,
                 ElectionId: electionDefinition.ballotHash,
                 BatchId: batchId,
-                CurrentSnapshotId: `${castVoteRecordId}-modified`,
-                UniqueId: ballotIdPrefix
-                  ? `${ballotIdPrefix}-${castVoteRecordId.toString()}`
-                  : castVoteRecordId.toString(),
+                CurrentSnapshotId: `${uniqueId}-original`,
+                UniqueId: uniqueId,
                 CVRSnapshot: [
                   {
                     '@type': 'CVR.CVRSnapshot',
-                    '@id': `${castVoteRecordId}-modified`,
+                    '@id': `${uniqueId}-original`,
                     ...buildCVRSnapshotBallotTypeMetadata(ballotType),
-                    Type: CVR.CVRType.Modified,
+                    Type: CVR.CVRType.Original,
```

Comment:
I made this change after noticing in `libs/backend` we only use the original snapshot for BMDs, but in this fixture-generator we use the `modified` snapshot. I don't think it has mattered, because later on we choose the snapshot via the `CurrentSnapshotId` key, so we find the existing snapshot.

Let me know if you think I should omit this change